### PR TITLE
Toggle: Fixed potential conflict between persist and state and fixed related documentation

### PR DESCRIPTION
**Pull request by pjackson28**

### CODE REVIEW

File: src/plugins/toggle/toggle-en.hbs [hunk 1]
```diff
@@ -86,6 +86,16 @@
 						</td>
 					</tr>
 					<tr>
+						<td><code>state</code> (v4.0.11+)</td>
+						<td>
+							Sets the initial state of a toggle element. Supports the following values:</td>
```
Comment (LaurentGoderre) [inline src/plugins/toggle/toggle-en.hbs]: remove the `</td>` to fix the build

File: src/plugins/toggle/toggle-fr.hbs [hunk 2]
```diff
@@ -29,6 +30,88 @@
...
+							<ul>
+								<li><strong>on:</strong> toggle element will only turn elements to the "on" toggle state.</li>
+								<li><strong>off:</strong> toggle element will only turn elements to the "off" toggle state.</li>
+							</ul>
+						</td>
+					</tr>
+					<tr>
+						<td><code>state</code> (v4.0.11+)</td>
+						<td>
+							Sets the initial state of a toggle element. Supports the following values:</td>
```
Comment (LaurentGoderre) [inline src/plugins/toggle/toggle-fr.hbs]: Remove the `</td>` to fix the build

File: src/plugins/toggle/toggle-en.hbs (hunk 1 above)
Comment (pjackson28) [inline src/plugins/toggle/toggle-en.hbs]: Fixed.

File: src/plugins/toggle/toggle-fr.hbs (hunk 2 above)
Comment (pjackson28) [inline src/plugins/toggle/toggle-fr.hbs]: Fixed.

**Comment by hsrudnicki** [comment]
@pjackson28 @LaurentGoderre : This issue is dependent on the others mentioned above, and just by occasion can this issue be closed as well?

**Comment by pjackson28** [comment]
@hsrudnicki This pull request was merged in back in April.

**Pull request merged**